uint8 uav_id
uint8 distribution

float64[] parameters

uint8 NONE=0
uint8 STATIC=1
uint8 UNIFORM=2
uint8 NORMAL=3
uint8 COUNT=4
